### PR TITLE
Add AirVisual Node/Pro docs

### PR DESCRIPTION
**Pull request by bachya**
## Proposed change

This PR adds docs for the forthcoming support of AirVisual Node/Pro units.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/32815
- This PR fixes or closes issue: N/A

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/airvisual.markdown
Comment (frenck) [inline source/_integrations/airvisual.markdown]:
```suggestion
Note that an API key-based entry can be mixed with one or more Node/Pro-based entries
```

File: source/_integrations/airvisual.markdown
Comment (frenck) [inline source/_integrations/airvisual.markdown]: Maybe just add 2 example blocks instead of adding it in as a code comment. It makes it more distinctive and easier to copy and paste.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by bachya** [comment]
Per [ADR-0010](https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md), 7a9b32773f8d1c15bd54f97e2eb0d74438f6f841 removes `configuration.yaml` docs for this new functionality.

**Comment by bachya** [comment]
@frenck Apologies: 7a9b327 re-triggered the need for a review, so I can't merge this.

**Comment by frenck** [comment]
Don't worry about it if this happens we pick it up, as the label/project board it moved to the parent-merged category 😉

**Review by frenck — APPROVED** [review]

**Pull request merged**
